openedx/core/djangoapps/discussions/tests/test_admin.py

```diff
@@ -0,0 +1,33 @@
+"""
+Tests that verify that the admin view loads.
+"""
+from django.test import TestCase
+from django.urls import reverse
+from django.test import Client
+from openedx.core.djangoapps.discussions.models import DiscussionsConfiguration, Provider
+from common.djangoapps.student.tests.factories import UserFactory
+
+
+class DiscussionsConfigurationAdminTest(TestCase):
+    """
+    Tests for discussion config admin
+    """
+    def setUp(self):
+        super().setUp()
+        self.superuser = UserFactory(is_staff=True, is_superuser=True)
```

Comment:
Cannot login to admin with the user returned from `SuperUserFactory()`. The user needs to be staff as well